### ISSUE TITLE
Compile Error

### ISSUE THREAD

**Issue by Ompha** [jobovy/extreme-deconvolution#17]
Hi Jo, 
While compiling, I got the following error 
```
src/bovy_randvec.c:17:10: fatal error: gsl/gsl_vector.h: No such file or directory
 #include <gsl/gsl_vector.h>
compilation terminated.
make: *** [src/bovy_randvec.o] Error 1
```

Can you please help me on this issue ? I wonder if this is a problem with my machine or the compilation?
Thanks. 
Cicero Lu

**Comment by jobovy** [comment]
Did you install the GSL? If not, see for example [these instructions](http://galpy.readthedocs.io/en/v1.3.0/installation.html#how-do-i-install-the-gsl). If yes, then the issue is that the installation cannot find the GSL, in that case [these instructions](http://galpy.readthedocs.io/en/v1.3.0/installation.html#the-galpy-installation-fails-because-of-c-compilation-errors) might help.

**Comment by Ompha** [comment]
Thank you for the reply. Upon installation, that problem went away but a new error pops up. 
```
gcc-8 -dynamiclib -W1 -single_module -o build/libextremedeconvolution.dylib -lm -lgsl -lgslcblas  \
	 -arch x86_64  \
	 src/bovy_isfin.o src/bovy_randvec.o src/calc_splitnmerge.o src/logsum.o src/minmax.o src/normalize_row.o src/proj_EM.o src/proj_EM_step.o src/proj_gauss_mixtures.o src/splitnmergegauss.o src/bovy_det.o\
	 src/proj_gauss_mixtures_IDL.o
gcc-8: error: unrecognized command line option '-W1'; did you mean '-W'?
make: *** [build/libextremedeconvolution.dylib] Error 1
```
Thank you for your help.

**Comment by jobovy** [comment]
Are you using the Makefile or the ``python setup.py`` installation? The Makefile doesn't have the ``W1`` option anywhere (``W1`` should be ``Wl``).

**Comment by Ompha** [comment]
This error is from Makefile. 
I was able to use python setup.py for installation(with several warning raised) but had trouble importing them as a module. The error is the following: 
```
OSError: dlopen(/anaconda/lib/python3.6/site-packages/extreme_deconvolution/_extreme_deconvolution.cpython-36m-darwin.so, 6): Symbol not found: _bovy_isfin
  Referenced from: /anaconda/lib/python3.6/site-packages/extreme_deconvolution/_extreme_deconvolution.cpython-36m-darwin.so
  Expected in: flat namespace
 in /anaconda/lib/python3.6/site-packages/extreme_deconvolution/_extreme_deconvolution.cpython-36m-darwin.so
```
The warning that seems to be related to the import issue during installation of setup.py is the  following: 
```
In file included from src/minmax.c:21:
src/proj_gauss_mixtures.h:53:13: warning: inline function 'bovy_isfin' is not
      defined [-Wundefined-inline]
inline bool bovy_isfin(double x); /* true if x is finite */
            ^
src/minmax.c:32:21: note: used here
        if (temp > *max && bovy_isfin(temp))
                           ^
1 warning generated.
```

Thank you very much!

**Comment by jobovy** [comment]
I think I see the problem: you are using the latest released version, but that version is >4 years old at this point. Please install the latest master version instead. The first issue you raised was fixed in #3 (please look at the issues before submitting a new one), the issue with the ``inline`` function was also fixed years ago.

**Comment by jobovy** [comment]
Please let me know if this is still posing issues, and if not please close the Issue here on the Issue Tracker.
